### PR TITLE
Hash parameter types that should be strings are now converted to a JSON string.

### PR DESCRIPTION
**Pull request by thinkingserious**
Please ensure the following steps have been run before submitting this pull request:

- [x] Run `make test` and ensure it passes locally
- [x] Run `make lint` to appease Rubocop

### CODE REVIEW

File: spec/integration/api/v2010/account/call_spec.rb [hunk 1]
```diff
@@ -71,6 +71,51 @@
     expect(actual).to_not eq(nil)
   end
 
+  it "receives create_with_twiml responses" do
```
Comment (childish-sambino) [inline spec/integration/api/v2010/account/call_spec.rb]: Don't think these changes should be here.

File: lib/twilio-ruby/rest/accounts/v1/credential/aws.rb [hunk 2]
```diff
@@ -254,7 +254,7 @@ def initialize(version, payload, sid: nil)
               @properties = {
                   'sid' => payload['sid'],
                   'account_sid' => payload['account_sid'],
-                  'friendly_name' => payload['friendly_name'],
+                  'friendly_name' => payload['friendly_name'].to_json,
```
Comment (childish-sambino) [inline lib/twilio-ruby/rest/accounts/v1/credential/aws.rb]: This is during deserialization of the API response. Shouldn't the fix be during serialization to generate the request payload?

File: spec/integration/api/v2010/account/call_spec.rb (hunk 1 above)
Comment (thinkingserious) [reply spec/integration/api/v2010/account/call_spec.rb]: Would this be because I did not pull in the latest changes from the `master` branch?

Comment (childish-sambino) [reply spec/integration/api/v2010/account/call_spec.rb]: More likely you're working off the latest api-defs and not what was last used for generation.

**Pull request closed without merging**